src/include/storage/storage_structure/disk_array.h

```diff
@@ -231,6 +231,14 @@ class InMemDiskArray : public BaseInMemDiskArray<T> {
     InMemDiskArray(FileHandle& fileHandle, StorageStructureID storageStructureID,
         common::page_idx_t headerPageIdx, BufferManager* bufferManager, WAL* wal);
 
+    static inline common::page_idx_t initializeHeaderPage(BMFileHandle& fileHandle,
```

Comment:
I would rename this to `addDAHPageToFile` or even `addDiskArrayToFile` and explain what it does in comments.

Reply:
Renamed to `addDAHPageToFile`.